lib/operator.js

```diff
@@ -192,11 +192,15 @@ proto._where = function(where) {
     const value = where[key];
     if (Array.isArray(value)) {
       wheres.push('?? IN (?)');
+    } else if (value === null) {
+      wheres.push('?? IS NULL');
     } else {
       wheres.push('?? = ?');
     }
     values.push(key);
-    values.push(value);
+    if (value !== null) {
```

Comment:
how about `undefined`?

Reply:
maybe replace '===' with '=='
or just not support undefined